src/providers/LocalVideoProvider/index.tsx

```diff
@@ -107,8 +107,22 @@ const LocalVideoProvider: React.FC = ({ children }) => {
   }, [audioVideo, tileId]);
 
   const value = useMemo(
-    () => ({ tileId, isVideoEnabled, setIsVideoEnabled, toggleVideo }),
-    [tileId, isVideoEnabled, setIsVideoEnabled, toggleVideo]
+    () => ({
+      tileId,
+      isVideoEnabled,
+      setIsVideoEnabled,
+      hasReachedVideoLimit,
+      setHasReachedVideoLimit,
+      toggleVideo,
+    }),
+    [
+      tileId,
+      isVideoEnabled,
+      setIsVideoEnabled,
+      hasReachedVideoLimit,
+      setHasReachedVideoLimit,
+      toggleVideo,
+    ]
```

Comment:
Need to update the localVideoContextType too.

Reply:
https://github.com/aws/amazon-chime-sdk-component-library-react/blob/main/src/types/index.ts#L29

Reply:
Good catch, updated it